schemas/blocks/list_collector.json

```diff
@@ -23,11 +23,20 @@
     "properties":{
       "item_title": {
           "$ref": "../string_interpolation/definitions.json#/string_with_placeholders"
+      },
+      "add_link_text": {
+        "type": "string"
+      },
+      "empty_list_text": {
+        "type": "string"
+      },
+      "title": {
+        "$ref": "../string_interpolation/definitions.json#/string_with_placeholders"
       }
     },
     "additionalProperties": false,
     "required": [
-        "item_title"
+        "item_title", "add_link_text", "empty_list_text", "title"
```

Comment:
add_link_text, empty_list_text aren't actually required?. They are only needed for section summary, ie using list collector without a section summary?.
Unless the intention is that list collectors will always use a summary?

Reply:
I don't think it is, given that summary isn't actually required itself. I've removed them for now.